### PR TITLE
Highlight linked review comment

### PR DESCRIPTION
**Pull request by shroudedcode**
#1164 for review comments.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ GitHub Enterprise is also supported. More info in the options.
 - Changes the default sort order of issues/PRs to "Recently updated"
 - [Makes the "Expand diff" button wider](https://user-images.githubusercontent.com/6978877/34470024-eee4f43e-ef20-11e7-9036-65094bd58960.PNG)
 - [Automatically closes modals when they’re no longer visible](https://user-images.githubusercontent.com/1402241/37022353-531c676e-2155-11e8-96cc-80d934bb22e0.gif)
-- [Highlights permalinked comments](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png)
+- [Highlights permalinked comments](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png) and [review comments](https://user-images.githubusercontent.com/29176678/37554360-7ff25bce-29d7-11e8-80d2-ecd9e25c42db.png)
```
Comment (bfred-it) [inline readme.md]: I don't think it's necessary to specify `review`, both are comments :)

**Comment by bfred-it** [comment]
Excellent! I just noticed it and there's already a PR for it :D

File: readme.md (hunk 1 above)
Comment (shroudedcode) [reply readme.md]: Yeah, removed it.

**Comment by shroudedcode** [comment]
@bfred-it What about this? Too much yellow/orange?

**Comment by bfred-it** [comment]
Looking good except the last color change, it wouldn't distinguish between "my" reactions and others'

screen shot 2018-03-23 at 23 30 54

Undone it since it's clearer with the original colors

**Pull request merged**